static/scss/results.scss

```diff
@@ -98,13 +97,26 @@ button.view-hide {
   margin-bottom: 2rem;
 }
 
+@media (max-width: 900px) {
```

Comment:
The media queries might be easier to read if they were in SCSS style, inside of the selector, instead of in proper CSS order.

Reply:
No worries, you’re just ahead of schedule.